### PR TITLE
FIxing acceptance tests

### PR DESCRIPTION
**Pull request by kobajagi**
Fixes issues with acceptance tests. List of changes:
- terraform SDK updated,
- disabled `MaxIndexCount` check for opensearch database resource (upstream bug),
- kafka version in tests updated,
- multiple import tests fixed.

### CODE REVIEW

File: exoscale/resource_exoscale_database_kafka_test.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 
 var (
 	testAccResourceDatabaseKafkaIPFilter = []string{"1.2.3.4/32"}
-	testAccResourceDatabaseKafkaVersion  = "3.1"
+	testAccResourceDatabaseKafkaVersion  = "3.2"
 	testAccResourceDatabasePlanKafka     = "business-4"
```
Comment (PhilippeChepy) [inline exoscale/resource_exoscale_database_kafka_test.go:18]: It could be nice if we can fix this value durably (maybe to be handled in a separate story).

**Review by PhilippeChepy — APPROVED** [review]
:rocket:

File: exoscale/resource_exoscale_database_kafka_test.go (hunk 1 above)
Comment (kobajagi) [reply exoscale/resource_exoscale_database_kafka_test.go:18]: I agree. Same applies for other databases so I'll create a separate story.

Comment (kobajagi) [reply exoscale/resource_exoscale_database_kafka_test.go:18]: Story created.

**Review by jessicatoscani — APPROVED** [review]

**Pull request merged**